utils/Azure/acrTools.ts

```diff
@@ -95,6 +107,25 @@ export async function sendRequestToRegistry(http_method: string, login_server: s
     assert(false, 'sendRequestToRegistry: Unexpected http method');
 }
 
+/** Sends a custom html request to a registry, and retrieves the image's digest.
+ * @param image : The targeted image.
+ */
+export async function getImageDigest(image: AzureImage): Promise<any> {
+    const { acrAccessToken } = await acquireACRAccessTokenFromRegistry(image.registry, `repository:${image.repository.name}:pull`);
+    let digest;
+    await ext.request.get('https://' + image.registry.loginServer + `/v2/${image.repository.name}/manifests/${image.tag}`, {
+        auth: {
+            bearer: acrAccessToken
+        },
+        headers: {
+            accept: 'application/vnd.docker.distribution.manifest.v2+json; 0.5, application/vnd.docker.distribution.manifest.list.v2+json; 0.6, application/vnd.oci.image.manifest.v1+json; 0.7'
+        }
+    }, (err, httpResponse, body) => {
+        digest = httpResponse.headers['docker-content-digest'];
```

Comment:
Do you need to handle err?

Reply:
We need to handle at least err. There neds to be something there so we can catch httpResponse. However I could have deleted the body, but I left it there because of debuging purposes. However I'm changing the name of the variables to start with an underscore so there is no confusion.